File: Examples/AspNetMvcCS/Controllers/DocumentViewerController.UsingPartial.cs
using System.Web.Mvc;
using GleamTech.DocumentUltimate.AspNet.UI;

namespace GleamTech.DocumentUltimateExamples.AspNetMvcCS.Controllers
{
    public partial class DocumentViewerController
    {
        public ActionResult UsingPartial()
        {
            var documentViewer = GetDocumentViewerModel();

            return View(documentViewer);
        }

        public ActionResult DocumentViewerPartialView()
        {
            var documentViewer = GetDocumentViewerModel();

            return PartialView(documentViewer);
        }

        private DocumentViewer GetDocumentViewerModel()
        {
            var documentViewer = new DocumentViewer
            {
                Width = 960,
                Height = 720,
                Resizable = true,
                Document = "~/App_Data/ExampleFiles/Default.pdf"
            };

            return documentViewer;
        }
    }
}

File: Examples/AspNetMvcVB/Controllers/DocumentViewerController.UsingPartial.vb
Imports GleamTech.DocumentUltimate.AspNet.UI

Namespace Controllers
    Partial Public Class  DocumentViewerController
        Inherits Controller

        Public Function UsingPartial() As ActionResult
            Dim documentViewer = GetDocumentViewerModel()

            Return View(documentViewer)
        End Function

        Public Function DocumentViewerPartialView() As ActionResult
            Dim documentViewer = GetDocumentViewerModel()

            Return PartialView(documentViewer)
        End Function

        Private Function GetDocumentViewerModel() As DocumentViewer
            Dim documentViewer = New DocumentViewer() With {
                .Width = 960,
                .Height = 720,
                .Resizable = True,
                .Document = "~/App_Data/ExampleFiles/Default.pdf"
            }

            Return documentViewer
        End Function
    End Class
End Namespace
